Frontend/src/Components/GeneralData/IconCell.js

```diff
@@ -0,0 +1,7 @@
+import { Box, Flex, Image} from "@chakra-ui/react";
```

Comment:
```suggestion
import { Box, Image} from "@chakra-ui/react";
```
To match suggestion below